depth_to_pointcloud/src/depth_to_pointcloud_node.cpp

```diff
@@ -69,7 +71,7 @@ static void depthCb(const sensor_msgs::msg::Image::SharedPtr image)
   g_pub_point_cloud->publish(cloud_msg);
 }
 
-void infoCb(sensor_msgs::msg::CameraInfo::SharedPtr info)
+static void infoCb(sensor_msgs::msg::CameraInfo::SharedPtr info)
```

Comment:
Mostly just to keep the functions/variables internal.  It can act as a hint to the compiler that it can do things like inline the functions, etc.  It's not really required, but doesn't hurt either.